packages/core/src/Accordion/Accordion.styled.tsx

```diff
@@ -52,7 +51,6 @@ export const StyledTrigger = styled(Accordion.Trigger).attrs(borderRadiusAttrs)`
   }
   border-top-left-radius: ${themeGet('borderRadii.lg')};
   border-top-right-radius: ${themeGet('borderRadii.lg')};
-  padding: 12px;
```

Comment:
Since we already published Accordion, these need to be configurable and use a default prop to prevent a breaking change.